config/locales/client.en.yml

```
@@ -2200,7 +2200,7 @@ en:
       edit_shared_draft: "Edit Shared Draft"
       title: "Or press %{modifier}Enter"
 
-      users_placeholder: "Add a user"
+      users_placeholder: "Add users/groups"
```

```suggestion
      users_placeholder: "Add users or groups"
```

Insanely nitpicky, but I think we should use this instead of the slash :)

---

Yeah, sounds good!